esp/scm/ws_topology.ecm

```diff
@@ -543,6 +543,7 @@ ESPrequest [nil_remove] TpLogFileRequest
     [min_ver("1.05")] int PageNumber(0);
     [min_ver("1.05")] bool LoadData(false);
     [min_ver("1.29")] bool IncludeLogFieldNames(true);
+    [min_ver("1.32")] string CompType;
```

Comment:
This will require changes to eclwatch (e.g. WsTopology.ts, tplog.xslt) for it to work.

Reply:
I haven't found a place where the CompType has to be set so far.